File: program335.cpp
/*
OUTPUT:
Enter the number:
1234
Number is:1234
4       3       2       1


Enter the number:
-2398
Number is:-2398
8       9       3       2

*/

#include<iostream>
using namespace std;

class Number
{
	private:
		int iNo;
		
	public:
	
		void Accept();
		void Display();
		void DisplayDigit();
};

void Number::Accept()
{
	cout<<"Enter the number:"<<endl;
	cin>>iNo;
}

void Number::Display()
{
	cout<<"Number is:"<<iNo<<endl;
	
}

void Number::DisplayDigit()
{
	int iDigit=0;
	
	if(iNo<0)
	{
		iNo=-iNo;
	}
	
	while(iNo!=0)
	{
		iDigit=iNo%10;
		cout<<iDigit<<"\t";
		iNo=iNo/10;
	}
}

int main()
{
	
	Number nobj;
	nobj.Accept();
	nobj.Display();
	nobj.DisplayDigit();
	
	
	return 0;
}

File: program335.java
import java.lang.*;
import java.util.*;

class Matrix
{
	public int iRow;
	public int iCol;
	public int Arr[][];
	
	public Matrix(int a,int b)
	{
		this.iRow=a;
		this.iCol=b;
		Arr=new int[iRow][iCol];
	}
	
	protected void finalize()
	{
		Arr=null;
		System.gc();
	}
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
		
		int i=0,j=0;
		
		System.out.println("Enter the number:");
		for(i=0;i<Arr.length;i++)
		{
			for(j=0;j<Arr[i].length;j++)
			{
				Arr[i][j]=sobj.nextInt();
			}
		}
		
		
	}
	
	public void Display()
	{
		System.out.println("Elements of array are:");
		
		int i=0,j=0;
		for(i=0;i<Arr.length;i++)
		{
			for(j=0;j<Arr[i].length;j++)
			{
				System.out.print(Arr[i][j]+"\t");
			}
			System.out.println();
		}
		
	}
		
}

class program335
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);
		System.out.println("Enter the number of rows:");
		int X=sobj.nextInt();
		
		System.out.println("Enter the number of columns:");
		int Y=sobj.nextInt();
		
		Matrix mobj=new Matrix(X,Y);
		mobj.Accept();
		mobj.Display();
	
	}
}
